orienteer-architect/src/main/java/org/orienteer/architect/util/OClassJsonDeserializer.java

```diff
@@ -0,0 +1,55 @@
+package org.orienteer.architect.util;
+
+import com.google.common.base.Strings;
+import com.google.common.collect.Lists;
+import com.google.gson.*;
+import com.orientechnologies.orient.core.metadata.schema.OType;
+
+import java.lang.reflect.Type;
+import java.util.List;
+
+/**
+ *
+ */
+public class OClassJsonDeserializer implements JsonDeserializer<List<OArchitectOClass>> {
```

Comment:
Coming back to the question about using of GSON. Is there ability to avoid it or provided functionality of this lib is pretty big and solid?